app/admin/tenants/onboarding.py

```diff
@@ -54,6 +54,9 @@ async def handle_tenant_update(
         if update.name:
             updated_actor["name"] = update.name
 
+        if update.group_id:
+            updated_actor["group_id"] = update.group_id
+
```

Comment:
The group id shouldn't be stored in the trust registry, this is something to query the tenants in the agents

Reply:
ah alright, thx - didn't know that. I'll change that then